### PR TITLE
Milestone 1: Project Setup

### PR DESCRIPTION
**Pull request by EL28DEV**
TicTacToe clone Milestone 1: Project Setup

In this milestone we:

-created 2 new branches
-made 1 branch as a default second as a feature branch
-added linters.yml and rubocop.yml
-added bin and lib folders which are includes files

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
# **STATUS: INVALID 🔴** 

**I am invalidating this CRR because the PR message is not descriptive. 
Kindly use this [lesson](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) to review how to write descriptive PRs along with messages.**

This Code Review Request will be marked as invalid in your Dashboard, so please submit another CRR once you are ready!

## Cheers!! 🍹 & Happy Coding.

## Reach me on slack for a chitchat 🆘  **Roy Mukuye**

**Comment by EL28DEV** [comment]
Thank you for your review. We added a descriptive message

**Review by mke2111 — CHANGES_REQUESTED** [review]
# **STATUS: CHANGES REQUIRED 🔴 ⚠️** 

## Hey @EL28DEV & @Mandeln, 👋🏽 
 **Thank you for the good job you got going on but the file is not yet executable.**

## REQUIRED CHANGES

- [ ] Kindly follow the instructions in your [lesson](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/) to make the main file executable. Line one of the `main` file should look like this ` #!/usr/bin/env ruby `.

**Kindly make all the requested changes and submit another CRR. 👍🏽** 

## Cheers!! 🍹 & Happy Coding.

# Reach me on slack for a chitchat 🆘  **Roy Mukuye**

File: bin/main [hunk 1]
```diff
@@ -1,3 +1,2 @@
-#!bin/main.rb
```
Comment (mke2111) [inline bin/main]: `#!/usr/bin/env ruby` this should be your line one

**Comment by EL28DEV** [comment]
Thank you! I think this time it is ok

**Review by SmithVyne — APPROVED** [review]
Approved 👍 👍 
Congratulations on applying all the changes required by the previous TSE. You can now merge this branch and proceed.

Reach out to me on Slack, **@smithvyne**, if you need answers to more questions.

**Pull request merged**